map/place_data.cpp

```diff
@@ -0,0 +1,133 @@
+#include "map/place_data.hpp"
+
+#include "indexer/cuisines.hpp"
+#include "indexer/ftypes_matcher.hpp"
+
+#include "platform/measurement_utils.hpp"
+
+#include "base/string_utils.hpp"
+
+#include "3party/opening_hours/opening_hours.hpp"
+
+namespace place_data
+{
+char const * kSubtitleSeparator = " • ";
+char const * kStarSymbol = "★";
+char const * kMountainSymbol = "▲";
+char const * kEmptyRatingSymbol = "-";
+char const * kPricingSymbol = "$";
+
+vector<string> GetLocalizedCuisines(string const & cuisineStr)
+{
+  vector<string> localized;
+  osm::Cuisines::Instance().ParseAndLocalize(cuisineStr, localized);
+  return localized;
+}
+
+string FormatStars(string const & starsStr)
+{
+  string stars;
+  int count = 0;
+  strings::to_int(starsStr, count);
+  CHECK(count >= 0, ("Star's count must be positive!"));
+  return string(count, *kStarSymbol);
```

Comment:
Sorry, I guess I gave you a bad piece of advice. As kStarSymbol is not a symbol, but UTF8-encoded string, you need to get back to old code:
```c++
string result;
for (int i = 0; i < count; ++i)
  result.append(kStarSymbol);
```

The same for all other usages or the strings above.